_episodes/04-performance.md

```diff
@@ -1,20 +1,20 @@
 ---
-title: "Teaching as a Performance Art"
```

Comment:
Improv.  ;)  

I'll note that we aren't always training faculty - we're often teaching novice instructors to whom the idea of teaching as performance may be a new idea.  

The language of performance (to me) implies that the audience matters more than the performer (not so much interactive vs. non-interactive), which I think is an important idea as well.  But if that can be conveyed in another way or with other language, I'm not tied to the word "performance" itself.